Core/src/org/sleuthkit/autopsy/actions/DeleteDataSourceAction.java

```diff
@@ -48,5 +51,13 @@ public void actionPerformed(ActionEvent event) {
             logger.log(Level.WARNING, "Error Deleting Data source " + selectedDataSource, e);
         }
     }
-
+    private static void deleteDataSource(Long dataSourceId) throws KeywordSearchServiceException {
```

Comment:
This method is not called by performAction. Perhaps a merge issue?

Reply:
Put it back in.